diff --git a/src/pam/Makefile.in b/src/pam/Makefile.in
index f4f066618..c335e81ec 100644
--- a/src/pam/Makefile.in
+++ b/src/pam/Makefile.in
@@ -31,7 +31,7 @@ LT_libs = \
 
 INSTALL_KAUTH = @INSTALL_KAUTH@
 
-MODULE_CFLAGS=${PAM_CFLAGS}
+MODULE_CFLAGS=${PAM_CFLAGS} -DBINDIR='"${bindir}"'
 
 all: test_pam pam_afs.la pam_afs.krb.la
 
diff --git a/src/pam/afs_util.h b/src/pam/afs_util.h
index 56a42457b..ac2b01ca5 100644
--- a/src/pam/afs_util.h
+++ b/src/pam/afs_util.h
@@ -24,9 +24,9 @@ extern int do_klog(const char *user, const char *password,
 		   const char *lifetime, const char *cell_name);
 extern afs_int32 getPAG(void);
 
-#define KLOG "/usr/afsws/bin/klog"
-#define KLOGKRB "/usr/afsws/bin/klog.krb"
-#define UNLOG "/usr/afsws/bin/unlog"
+#define KLOG BINDIR "/klog"
+#define KLOGKRB BINDIR "/klog.krb"
+#define UNLOG BINDIR "/unlog"
 #define IGNORE_MAX 1000
 
 #if	defined(AFS_HPUX_ENV)
