diff --git a/src/config/Makefile.config.in b/src/config/Makefile.config.in
index 90a47f3ac..9de6c29a4 100644
--- a/src/config/Makefile.config.in
+++ b/src/config/Makefile.config.in
@@ -218,11 +218,11 @@ COMPILE_ET_C=${COMPILE_ET} -emit c
 COMMON_INCL=$(MODULE_INCLUDE) -I${TOP_OBJDIR}/src/config -I${TOP_INCDIR} \
 	    -I${srcdir} -I. 
 
-COMMON_CFLAGS=$(CFLAGS) ${DBG} ${OPTMZ} $(XCFLAGS) \
+COMMON_CFLAGS=$(CFLAGS) $(XCFLAGS) \
 	      ${COMMON_INCL} ${INCLUDE_LIBINTL} \
 	      $(CPPFLAGS_roken) $(ARCHFLAGS)
 
-COMMON_LDFLAGS=$(LDFLAGS) $(LDFLAGS_hcrypto) $(LDFLAGS_roken) $(DBG) $(OPTMZ)
+COMMON_LDFLAGS=$(LDFLAGS) $(LDFLAGS_hcrypto) $(LDFLAGS_roken)
 
 # LWP Flags
 LWP_CFLAGS=$(MODULE_CFLAGS) $(COMMON_CFLAGS)
